src/test/pythonEnvironments/discovery/locators/virtualenvwrapperLocator.unit.test.ts

```diff
@@ -32,7 +31,7 @@ suite('Virtualenvwrapper Locator Tests', () => {
         getDefaultDirStub.restore();
     });
 
-    test('WORKON_HOME is not set, and the interpreter is is in a subfolder', async () => {
+    test('WORKON_HOME is not set, and the interpreter is is in a subfolder of virtualenvwrapper', async () => {
```

Comment:
```suggestion
    test('WORKON_HOME is not set, and the interpreter is in a subfolder of virtualenvwrapper', async () => {
```
Not blocking, just a typo.

Reply:
can i put this one on my faulty keyboard? just disregard the fact that it got recently replaced and you know, it's a word..... 😅